.github/workflows/appimage.yml

```diff
@@ -1,8 +1,16 @@
-name: Build AppImage SLAM Server
+name: Build AppImage Cartographer
 
 on:
   workflow_dispatch:
+    inputs:
+      release_type:
+        required: true
+        type: string
```

Comment:
I'd suggest setting a default and a description here, so it's clear what this is when someone runs it manually.

Reply:
Done!

Reply:
@Otterverse since we require these inputs, the default is never used:

```
.github/workflows/appimage.yml:17:18: input "release_type" of workflow_call event has the default value "latest", but it is also required. if an input is marked as required, its default value will never be used [events]
   |
17 |         default: latest
```

Any option on if we want to remove the required flag or the default?

Reply:
default pre-populates the field in the UI on github for human use, even if it's also required (that just means they can't leave it blank.) I definitely say to keep both.